### PR TITLE
Add filter suggestions

### PR DESCRIPTION
**Pull request by TristanCacqueray**

### CODE REVIEW

File: monocle/db/db.py [hunk 1]
```diff
@@ -385,12 +385,12 @@ def __init__(
             "self_merged": {"type": "boolean"},
         }
         self.mapping.update(self.new_fields)
-        settings = {"mappings": {"properties": self.mapping}}
         self.ic = self.es.indices
         if create:
             try:
                 self.ic.get(index=self.index)
             except NotFoundError:
+                settings = {"mappings": {"properties": self.mapping}}
```
Comment (TristanCacqueray) [inline monocle/db/db.py:393]: nit: that diff does not seem related.

**Review by TristanCacqueray — COMMENTED** [review]
LGTM

File: monocle/db/db.py (hunk 1 above)
Comment (morucci) [reply monocle/db/db.py:393]: I'll move it.

**Comment by morucci** [comment]
Does it work as expected from your side ? I no longer get Change state choice listing, neither project and approval.

**Comment by TristanCacqueray** [comment]
There is indeed an issue when the filtered list is smaller than the limit: the take function from Belt.List returns a nothing. I'll fix that.

**Comment by TristanCacqueray** [comment]
So a231c61 fixes the empty list issue, and  54ae78d fixes a bug where the create value was buggy.

**Review by morucci — APPROVED** [review]
